lib/OrganizationsFieldArray/OrganizationCard.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AppIcon } from '@folio/stripes/core';
+import { FormattedMessage } from 'react-intl';
+import { get, isEmpty } from 'lodash';
+import {
+  Card,
+  Icon,
+  MultiColumnList,
+  Layout,
+} from '@folio/stripes/components';
+import css from './OrganizationCard.css';
+
+export default class OrganizationCard extends React.Component {
+  static propTypes = {
+    cardStyle: PropTypes.string,
+    headerStart: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.arrayOf(PropTypes.node),
+    ]),
+    headerEnd: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.arrayOf(PropTypes.node),
+    ]),
+    id: PropTypes.string,
+    interfaces: PropTypes.object,
+    meta: PropTypes.shape({
+      error: PropTypes.node,
+    }),
+    org: PropTypes.object,
+    renderDefault: PropTypes.func,
+  };
+
+  renderError = (error) => (
+    <Layout className={`textCentered ${css.error}`}>
+      <strong>
+        {error}
+      </strong>
+    </Layout>
+  );
+
+  renderInterfaces = (interfaces) => {
+    return interfaces.length ? (
+      <MultiColumnList
+        columnMapping={{
+          name: <FormattedMessage id="stripes-erm-components.interface.name" />,
+          username: <FormattedMessage id="stripes-erm-components.interface.username" />,
+          password: <FormattedMessage id="stripes-erm-components.interface.password" />,
+          type: <FormattedMessage id="stripes-erm-components.interface.type" />,
+          notes: <FormattedMessage id="stripes-erm-components.interface.notes" />,
+        }}
+        columnWidths={{
+          name: 150,
+          notes: 250,
+          username: 130,
+          password: 130,
+          type: 150,
+        }}
+        contentData={interfaces}
+        formatter={{
+          name: item => (
+            <span>
+              {item.name}
+              <a href={item.uri}>
+                <Icon icon="external-link" iconPosition="end" />
+              </a>
+            </span>
+          ),
+          notes: item => get(item, 'notes'),
+          username: item => get(item, 'username'),
+          password: item => get(item, 'password'),
+          type: item => get(item, 'type'),
```

Comment:
[`type` is an array](https://s3.amazonaws.com/foliodocs/api/mod-organizations-storage/interface.html#organizations_storage_interfaces__id__get)